packages/flutter_tools/lib/src/base/analyze_size.dart

```diff
@@ -0,0 +1,370 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_tools/src/base/process.dart';
+import 'package:flutter_tools/src/base/terminal.dart';
```

Comment:
Do these need to be relative imports to fix the analyze bot?

Reply:
Looks like other files have relative imports! Changed to relative imports.